cms/templates/course_outline.html

```diff
@@ -50,6 +50,40 @@ <h2 class="title title-3">${_("This course was created as a re-run. Some manual
     </div>
   </div>
   %endif
+
+  %if ora1_status.get('ora1_components') or ora1_status.get('ora1_advance_modules'):
+  <div class="wrapper wrapper-alert wrapper-alert-error is-shown">
+    <div class="alert announcement">
+      <i class="feedback-symbol fa fa-bullhorn"></i>
+
+      <div class="copy">
+        <h2 class="title title-3">${_("This course uses ORA1. ORA1 has been deprecated and removed from edX.")}</h2>
+
+        <p>${_("Assignments using ORA1 will not be accessible to students. Please replace any ORA1 assignments in this course with ORA2 assignments.")}</p>
+
+
+      %if ora1_status.get('ora1_components'):
+          <div>
+              <nav class="nav-related" aria-label="${_('ORA 1 Components')}">
+                  <ul>
+                      % for component_parent_key, component_display_name in ora1_status['ora1_components']:
+                        <li class="nav-item"><a href="${component_parent_key}">${_(component_display_name)}</a></li>
+                      % endfor
+                  </ul> 
+              </nav>
+          </div>
+      %endif
+
+       <p>
+        ${_("You also need to remove peergrading and combinedopenended from the \"Advanced Module List\" in ")}
+        <a href="http://studio.tnl2303.m.sandbox.edx.org/settings/advanced/edX/DemoX/Demo_Course">Studio Advanced Settings</a>.
```

Comment:
@srpearce There are three pieces of text.
* The heading: this will be visible whenever this box is visible.
* The message with the list of ora1 components: this will be visible if the course has any ora1 components.
* The message with the link to the advance settings page: this will be visible if either `peergrading` or `combinedopenended` are the in the advance modules list.

Can you please suggest the appropriate wording. You can look at how these appear on http://studio.tnl2303.m.sandbox.edx.org/course/edX/DemoX/Demo_Course.